examples/xusb.c

```diff
@@ -1199,7 +1199,7 @@ int main(int argc, char** argv)
 		printf("   -l lang : language to report errors in (ISO 639-1)\n");
 		printf("   -w      : force the use of device requests when querying WCID descriptors\n");
 		printf("If only the vid:pid is provided, xusb attempts to run the most appropriate test\n");
-		return 0;
+		return EXIT_SUCCESS;
```

Comment:
How about this one? I feel `show_help` should return `EXIT_SUCCESS` and `(argc == 1) || (argc > 7)` has to be `EXIT_FAILURE`. Is this expected by most users?